reports/report_builder.py

```diff
@@ -1,6 +1,7 @@
 # Python
 import io
 from collections import Counter, OrderedDict
+from itertools import count
```

Comment:
I don't think we are using this.

Reply:
No we're not... I'm blaming VS Code auto-importer for that 😎